apps/heft/src/cli/actions/HeftActionBase.ts

```diff
@@ -128,7 +128,7 @@ export abstract class HeftActionBase extends CommandLineAction {
       encounteredError = true;
       throw e;
     } finally {
-      this.recordMetrics();
+      this.recordMetrics(encounteredError);
```

Comment:
Seems a little premature to report this? On 137 the value is changed, and it seems the remaining code trusts it more.